Quadriga/src/main/java/edu/asu/spring/quadriga/service/transformation/ITransformationManager.java

```diff
@@ -17,4 +20,17 @@
 
     public List<TransformFilesDTO> getTransformationsList();
 
+    /**
+     * Retrieve the absolute file paths of patternFile and MappingFile as an
+     * arraylist of strings.
+     * 
+     * @param transformationId
+     *            Transformation ID of the transformatioin.
+     * @return Returns the absolute file paths in an arraylist of Strings
+     * @throws FileNotFoundException
+     * @throws IOException
+     * @throws FileStorageException
+     */
+    ArrayList<String> retrieveTransformationFilePaths(String transformationId) throws FileStorageException;
```

Comment:
If at all, this should be ```List``` not ```ArrayList```. But I'm sure there is a better solution that returning a list of strings.